derive_builder/Cargo.toml

```diff
@@ -34,7 +34,7 @@ log = { version = "0.3", optional = true }
 env_logger = { version = "0.4", optional = true }
 derive_builder_core = { version = "0.1", path = "../derive_builder_core" }
 skeptic = { version = "0.9", optional = true }
-compiletest_rs = { version = "0.2", optional = true }
+compiletest_rs = { version = "0.3.3", optional = true }
 
```

Comment:
I was hitting <https://github.com/laumann/compiletest-rs/issues/86> before updating the version.